packages/react-i18n/src/tests/i18n.test.ts

```diff
@@ -1277,6 +1277,163 @@ describe('I18n', () => {
+            time: '6:34 PM',
+          },
+        },
+        defaultTranslations,
+        i18n.locale,
+      );
+    });
+
+    it('formats a date/time in spanish', () => {
+      const date = new Date('2022-06-12T16:34:56Z');
+      const i18n = new I18n(defaultTranslations, {
+        locale: 'es',
+        country: 'ES',
+      });
+
+      i18n.formatDateTime(date, {
+        month: 'short',
+        day: 'numeric',
+        year: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        timeZone: timezone,
+      });
+
+      expect(translate).toHaveBeenCalledWith(
+        'date.humanize.datetime',
+        {
+          pseudotranslate: false,
+          replacements: {
+            date: '12 jun 2022',
+            time: '18:34',
+          },
+        },
+        defaultTranslations,
+        i18n.locale,
+      );
+    });
+
+    it('formats a date/time in french', () => {
+      const date = new Date('2022-06-12T16:34:56Z');
+      const i18n = new I18n(defaultTranslations, {
+        locale: 'fr',
+        country: 'FR',
+      });
+
+      i18n.formatDateTime(date, {
+        month: 'short',
+        day: 'numeric',
+        year: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        timeZone: timezone,
+      });
+
+      expect(translate).toHaveBeenCalledWith(
+        'date.humanize.datetime',
+        {
+          pseudotranslate: false,
+          replacements: {
+            date: '12 juin 2022',
+            time: '18:34',
+          },
+        },
+        defaultTranslations,
+        i18n.locale,
+      );
+    });
+
+    it('formats a date/time in dutch', () => {
+      const date = new Date('2018-01-01T16:34:56Z');
+      const i18n = new I18n(defaultTranslations, {
+        locale: 'nl',
+        country: 'NL',
+      });
+
+      i18n.formatDateTime(date, {
+        month: 'short',
+        day: 'numeric',
+        year: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        timeZone: timezone,
+      });
+
+      expect(translate).toHaveBeenCalledWith(
+        'date.humanize.datetime',
+        {
+          pseudotranslate: false,
+          replacements: {
+            date: '1 jan. 2018',
+            time: '17:34',
+          },
+        },
+        defaultTranslations,
+        i18n.locale,
+      );
+    });
+
+    it('formats a date/time in english', () => {
+      const date = new Date('2018-01-01T12:34:56Z');
+
+      const i18n = new I18n(defaultTranslations, {
+        ...defaultDetails,
+        timezone,
+        locale: 'en',
+        country: 'GB',
+      });
+
+      i18n.formatDateTime(date, {
+        month: 'short',
+        day: 'numeric',
+        year: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        hour12: false,
+      });
+
+      expect(translate).toHaveBeenCalledWith(
+        'date.humanize.datetime',
+        {
+          pseudotranslate: false,
+          replacements: {
+            date: 'Jan 1, 2018',
```

Comment:
As far as I know the date format in the UK is also `day-month-year`.

Reply:
Fixed the test to use the right locale